src/platforms/javascript/common/troubleshooting/index.mdx

```diff
@@ -427,3 +427,29 @@ Remember to pass in `true` as the second parameter to `addEventListener()`. With
 Starting with version 7.0.0, the Sentry JavaScript SDK uses ES6 syntax, along with a few other ES6+ language features, like object spread. If you are down-compiling your code in order to target older browsers that don't support such syntax, you'll need to include the Sentry SDK in that process.
 
 <PlatformContent includePath="troubleshooting/older-browser-support" />
+
+## Build Errors with Vite
+
+If you are using the [Vite Bundler](https://vitejs.dev/) and a Sentry NPM package and you get an error like so:
+
+```
+Error: Could not resolve './{}.js' from node_modules/@sentry/utils/esm/index.js
+```
+
+This might be because [`define` option in your Vite config](https://vitejs.dev/config/shared-options.html#define) is string replacing some variable used by the Sentry SDK, like `global`, causing build errors. Vite recommends using `define` for CONSTANTS only, and not putting `process` or `global` into the options. To fix this build error, remove or update your `define` option functionality, like what is shown below:
```

Comment:
```suggestion
This might be because the [`define` option in your Vite config](https://vitejs.dev/config/shared-options.html#define) is string-replacing some variable used by the Sentry SDK, like `global`, causing build errors. Vite recommends using `define` for CONSTANTS only, and not putting `process` or `global` into the options. To fix this build error, remove or update your `define` option, as shown below:
```